product_supplierinfo_discount/__openerp__.py

```diff
@@ -22,9 +22,10 @@
 
 {
     "name": "Discounts in product supplier info",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "author": "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```

Comment:
Please switch these two companies (Serv. Tecn... and Antiun) to the new one we have made: `Tecnativa`